database.js

```diff
@@ -0,0 +1,51 @@
+let express = require("express");
```

Comment:
Can we move database.js to app/
Usually we want to keep any source code in app/